infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/runtime/LogReplicationClientRouter.java

```diff
@@ -480,6 +480,15 @@ public synchronized void onError(Throwable t) {
         runtimeFSM.input(new LogReplicationRuntimeEvent(LogReplicationRuntimeEventType.ERROR, t));
     }
 
+    /**
+     * Cluster Change Callback.
+     *
+     * @param clusterDescriptor remote cluster descriptor
+     */
+    public synchronized void onClusterChange(ClusterDescriptor clusterDescriptor) {
+        channelAdapter.clusterChangeNotification(clusterDescriptor);
```

Comment:
should we add a log? this is not expected to happen frequently, but I think it would be nice to be able to know when it happened..

Reply:
Yes, Added a warning log.